apps/google-analytics/src/index.tsx

```diff
@@ -84,25 +101,37 @@ export class SidebarExtension extends React.Component<
       };
 
       return (
+        <>
         <div
           ref={renderAuthButton}
           className={isAuthorized ? styles.hidden : styles.signInButton}
         />
+        {helpTextFrag}
+        </>
       );
     }
 
+    if (!isContentTypeConfigured) {
+      return <Paragraph className={styles.lightText}>The {contentTypeName}{' '}
+      content type hasn&apos;t been configured for use with this app. It must
+      have a field of type short text and must be added to the list of
+      content types in this app&apos;s configuration.</Paragraph>;
```

Comment:
How about we add a deeplink to the app config screen?

Reply:
currently not possible